internal/kojiapi/server.go

```diff
@@ -117,7 +120,13 @@ func (server *Server) PostCompose(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "Only single-image composes are currently supported", http.StatusBadRequest)
 		return
 	}
-	id, err := server.workers.Enqueue(ir.manifest, nil)
+	id, err := server.workers.Enqueue(ir.manifest, []*target.Target{
+		target.NewKojiTarget(&target.KojiTargetOptions{
+			Filename:        ir.filename,
+			UploadDirectory: "osbuild-composer-koji-" + uuid.New().String(),
```

Comment:
The worker has to give koji a unique upload directory?

Reply:
That's my understanding, yes. To this and all the other API questions: I have many of the same, and think we should reach back out to Tomas and friends once we have this up and running so we can fill it all in correctly.